test/config/integration/server.yaml

```diff
@@ -7,167 +7,217 @@ static_resources:
     filter_chains:
     - filters:
       - name: envoy.http_connection_manager
-        config:
-          value:
-            drain_timeout_ms: 5000
-            route_config:
-              virtual_hosts:
-              - require_ssl: all
-                routes:
-                - cluster: cluster_1
-                  prefix: "/"
-                domains:
-                - www.redirect.com
-                name: redirect
-              - routes:
-                - prefix: "/"
+        typed_config:
```

Comment:
I'm a little confused why all this churn was needed. Isn't it as simple as just specifying the type param in various places as was done elsewhere? I must be missing something?

Reply:
The places where we switch to `load_assignment` make sense to me, but this particular file has a lot of churn. Most of this seems legit, but it would be helpful if @moderation could walk reviewers through this.